benchmarks/Microsoft.AspNetCore.Routing.Performance/Matchers/MatcherBuilder.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Routing.Matchers
+{
+    internal abstract class MatcherBuilder
```

Comment:
This is just here to make it easy and uniform to write benchmarks, I don't think there's any value in trying to put this in the product.